Sources/StreamChatUI/ChatMessageList/MessageComposer/ChatMessageComposerMentionCellView_Tests.swift

```diff
@@ -0,0 +1,236 @@
+//
+// Copyright © 2021 Stream.io Inc. All rights reserved.
+//
+
+import StreamChat
+import StreamChatTestTools
+@testable import StreamChatUI
+import XCTest
+
+class ChatMessageComposerMentionCellView_Tests: XCTestCase {
+    /// Default reference width for the cell on current tested device
+    private static var defaultCellWidth = UIScreen.main.bounds.width - 16
```

Comment:
The goal for the tests is to be isolated and reproducible. This is the exact opposite of this. Just replace it with a constant and it will be ok 👍

Reply:
Done :)